code/modules/projectiles/guns/energy/kinetic_accelerator.dm

```diff
@@ -33,10 +33,10 @@
 			var/obj/item/borg/upgrade/modkit/M = A
 			. += "<span class='notice'>There is \a [M] installed, using <b>[M.cost]%</b> capacity.</span>"
 
-/obj/item/gun/energy/kinetic_accelerator/crowbar_act(mob/living/user, obj/item/I)
+/obj/item/gun/energy/kinetic_accelerator/screwdriver_act(mob/living/user, obj/item/I)
 	. = TRUE
 	if(modkits.len)
-		to_chat(user, "<span class='notice'>You pry the modifications out.</span>")
+		to_chat(user, "<span class='notice'>You unscrew the modifications out.</span>")
```

Comment:
Weird nitpick, but "You unscrew the modifications out." reads somewhat awkwardly. I don't think "to unscrew out" is common usage. Maybe "You unfasten the modifications." or something along those lines?

Reply:
unfasten is the right word